tests/acceptance/job/basic-layout-test.js

```diff
@@ -58,7 +58,7 @@ test('visiting job-view', function (assert) {
     assert.equal(jobPage.createdBy.text, 'Mr T');
     assert.ok(jobPage.createdBy.avatarSrc.startsWith('/images/favicon-gray.png'));
 
-    assert.equal(jobPage.log, 'Hello log');
+    assert.equal(jobPage.log, '1Hello log');
```

Comment:
Same here, do you know what have caused this change? It may also affect our customers, which may be unexpected.